mxcubecore/HardwareObjects/abstract/sample_changer/Crims.py

```diff
@@ -16,6 +16,8 @@ def get_image(url: str) -> Optional[bytes]:
         logging.getLogger("user_level_log").warning(
             "Failed to fetch image from %s: %s", url, e
         )
+
+        logging.getLogger("HWR").exception("")
```

Comment:
I write it only here but I guess this applies to all hardware objects, right?

```suggestion
        self.log.exception("")
```

Reply:
That will be done in a second pass as we are not sure that all the statements belong to a HardwareObject ...